pkg/slices/slices_test.go

```diff
@@ -171,9 +193,18 @@ func BenchmarkUnique(b *testing.B) {
 				rand.Shuffle(len(orig), func(i, j int) {
 					orig[i], orig[j] = orig[j], orig[i]
 				})
-				b.StartTimer()
 
-				Unique(values)
+				key := func(i int) int {
+					return values[i]
+				}
```

Comment:
Nit: since the key function won't change, It's possible to move it before the loop, just after `values` declaration and before the timer reset.

Reply:
Fixed in the latest push. I didn't update the benchmark result in the commit message since the change didn't make much difference.